src/main/java/org/reactome/release/qa/common/SkipList.java

```diff
@@ -0,0 +1,78 @@
+package org.reactome.release.qa.common;
+
+import org.gk.model.GKInstance;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SkipList {
+
+    public List<String> skipList = new ArrayList<>();
+
+    /**
+     * The SkipList class contains functions for reading in an assumed skiplist, and checking its contents.
+     * The main assumption is that the skiplist file will be stored in a "resources" folder, which needs to be located
+     * in the same directory as the executor, and be in the format "step_name_skip_list.txt".
+     * @param displayName - String, displayName of class calling SkipList class.
+     * @throws IOException, thrown when file does not exist. Note: Does not terminate program.
+     */
+    public SkipList(String displayName) throws IOException {
+        if (Files.exists(getSkipListFilePath(displayName))) {
+            skipList = readDbIdsFromSkipListFile(getSkipListFilePath(displayName));
+        } else {
+            FileNotFoundException e = new FileNotFoundException();
```

Comment:
Can this exception be thrown instead instead of outputting the stacktrace?

The caller creating a new skip list is probably best to catch the exception and print the stack trace there.

Reply:
It is also important to throw here to be in sync with the JavaDoc.

Reply:
Done - It now throws the Exception here, and I added try/catches to each of the modules that call it